src/hooks/HltbApi.ts

```diff
@@ -29,10 +30,16 @@ async function fetchSearchKey() {
                     if (scriptResponse.status === 200) {
                         const scriptText = await scriptResponse.text();
                         const pattern =
-                            /\/api\/lookup\/"\.concat\("([a-zA-Z0-9]+)"\)\.concat\("([a-zA-Z0-9]+)"\)/;
+                            /\/api\/[a-zA-Z]*\/"\.concat\("([a-zA-Z0-9]+)"\)\.concat\("([a-zA-Z0-9]+)"\)/;
```

Comment:
May I ask you to split fixes in multiple PRs if that's okay?
IMO it is better to keep things separated logically - one PR per one bug or feature.